server/app/services/geo/esri/RealEsriClient.java

```diff
@@ -320,11 +330,20 @@ CompletionStage<Optional<JsonNode>> fetchServiceAreaFeatures(
     return tryRequest(request, this.ESRI_EXTERNAL_CALL_TRIES)
         .thenApply(
             res -> {
+              incrementEsriEndpointUsageCounter(res.getUri());
+
               // return empty if still failing after retries
               if (res.getStatus() != 200) {
                 return Optional.empty();
               }
               return Optional.of(res.getBody(json()));
             });
   }
+
+  /** Increment the counter for esri endpoint usage */
+  private void incrementEsriEndpointUsageCounter(URI uri) {
+    // Ignores query parameters and fragments.
+    String endpointUrl = String.format("%s://%s%s", uri.getScheme(), uri.getHost(), uri.getPath());
```

Comment:
Does getPath start with a "/" or should that be added to the format?

Reply:
It already has it